root/src/main/java/com/example/servingwebcontent/ConnectDatabase.java

```diff
@@ -15,21 +15,25 @@ public ConnectDatabase (String database, String user, String pass) {
 		}
 	}
 
-	public ResultSet getMovie (String id) {
+	public Movie getMovie (String id) {
 		ResultSet rs = null;
 		String query = "select * from movies where id=?";
+		Movie movie = null;
 		try {
 			PreparedStatement stmt = con.prepareStatement(query);
 			stmt.setString(1, id);
 			rs = stmt.executeQuery();
+			if (rs.next()) {
+				movie = new Movie(rs.getString("id"), rs.getString("type"), rs.getString("name"), rs.getString("releasedate"), rs.getInt("duration"), rs.getString("filmrating"));
```

Comment:
it's common practice to use static final Strings for constants. This makes it so if you change the string, you only change it in one place and not everywhere.

For example:

public static final String ID = "id";

You would need to add it to the correct file so it would have the correct context. For Movie constants, you can put it in the Movie object for now.

Reply:
I'm not sure I understand. Should I define the strings in the movie object? How would I be able to grab the string values from the movie object if the object hasn't been created yet?